src/main/java/it/pagopa/interop/probing/telemetry/writer/dto/TelemetryDto.java

```diff
@@ -0,0 +1,19 @@
+package it.pagopa.interop.probing.telemetry.writer.dto;
+
+import it.pagopa.interop.probing.telemetry.writer.util.EserviceStatus;
+import lombok.Data;
+
+@Data
+public class TelemetryDto {
+
+  private String serviceName;
```

Comment:
```suggestion
  private UUID eserviceId;

  private UUID versionId;
```
or
```suggestion
  private UUID eserviceRecordId;
```
based on which key you need to use